nixos/modules/services/cluster/spark/default.nix

```diff
@@ -0,0 +1,151 @@
+{config, pkgs, lib, ...}:
+let
+  cfg = config.services.spark;
+in
+with lib;
+{
+  options = {
+    services.spark = {
+      master = {
+        enable = mkEnableOption "Enable spark master service";
```

Comment:
The word "Enable" is redundant here. See definition of `mkEnableOption`.

Reply:
Fixed, thanks!